src/odemis/dataio/tiff.py

```diff
@@ -1221,13 +1224,20 @@ def _findImageGroups(das):
             or prev_da.metadata.get(model.MD_ROTATION, 0) != da.metadata.get(model.MD_ROTATION, 0)
             or prev_da.metadata.get(model.MD_SHEAR, 0) != da.metadata.get(model.MD_SHEAR, 0)
            ):
-            # new group
-            group_ifd = current_ifd
+            # don't add a new group if passed group check and it's z-stacks
+            if prev_da.metadata.get(model.MD_DIMS, None) == "ZYX" and dims == "ZYX" and da.ndim == 3 and prev_da.ndim == 3:
```

Comment:
I think it's overly catching cases. Essentially, it's now merging any DataArray which is of shape ZYX. Including possibly SEM data, or ZYX of different sizes, or ZYX at different positions in XY.

I would suggest to just modify the original condition to be more specific and say "if previous image and current image both have C dimension == 1", which the default C dimension being of length 1. 
You can use something like this to read the C dimension:
```
        dims = da.metadata.get(model.MD_DIMS, "CTZYX"[-da.ndim::])
        len_c = 1
        if "C" in dims:
            len_c = da.shape[dims.index("C")]
```